### ISSUE TITLE
Accessibility Label for Edit Orders on Order List is Hidden

### ISSUE THREAD

**Issue by jage9** [lat9/edit_orders#195]
This code, which appears on the main order list page, is using an aria-hidden="true" property. This hides the title of "Edit this Order" from screen reader users.

I.E.

<a href="https://www.mystore.com/store/myadmin/index.php?cmd=edit_orders&amp;selected_box=customers&amp;status=2&amp;page=1&amp;oID=24060&amp;action=edit" class="btn btn-default btn-edit"><i class="fa fa-wrench overlay" aria-hidden="true" title="Edit this Order"></i></a>

Is aria-hidden used for another reason       in this situation?

Proposed fix:

Remove
aria-hidden="true"
from this line. I tested in the latest Chrome, and the Edit this Order text is properly exposed.

**Comment by lat9** [comment]
@jage9, I'm not seeing that aria-hidden property on that edit order button.  Are you using the files currently present in this repository?

**Comment by jage9** [comment]
Yes, exactly it. And I didn't even realize there were nav buttons on the top, so glad those got fixed as well.

**Comment by lat9** [comment]
Tag added (great idea).  I'd prefer to keep this issue open until the 4.6.0 release since it's got code staged for it.

For the other observations, they all sound like a good idea.  

The "Edit" icon on the order's listing is visually different from the "Edit" button at the bottom of the order-details display, since it's an icon rather than a button.
